### PR TITLE
Query timer bug release note

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: blog-service/2023-09-08-search.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - search
+  - query
+  - elapsed timer
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+Update - We've enhanced the elapsed query timer so that now everytime you switch the sumo internal tab and come back the timer does not starts again from 0, instead, it shows the steady elapsed time.
```
Comment (sukhhanda1) [inline blog-service/2023-09-08-search.md:14]: Do we also need to highlight the issue where our clock run very slow when browser is minimized or some other window in focused instead of the browser? This is because https://usefulangle.com/web-updates/post/110/chrome-88-will-throttle-timers-1-minute

**Review by sukhhanda1 — APPROVED** [review]
LGTM

**Comment by sukhhanda1** [comment]
> LGTM

Looks Good To Me

File: blog-service/2023-09-08-search.md (hunk 1 above)
Comment (manashar4) [reply blog-service/2023-09-08-search.md]: I would look to reword this as ----

We have identified an issue with the elapsed query timer, in scenarios where a user switches between SUMO internal tabs when a query is in progress. This bug resulted in inaccurate elapsed query time calculations. With this fix, the elapsed query time will be updated accurately.

Comment (JV0812) [reply blog-service/2023-09-08-search.md]: @manashar4 Usually, in release note we will not mention the cause of the issue instead we just mention the solution and how it is going to help the user. Also, the content above have some passive voice issue. I have updated the content with more clarity now, can you please check.

Comment (manashar4) [reply blog-service/2023-09-08-search.md]: @JV0812 - I feel that makes sense for Features but with Bugs i dont know if that is a good approach. 
In any case, just one minor edit suggestion

We've enhanced the elapsed query timer so that while a query is running and _you_ **users** navigate to a different Sumo internal tab or browser tab and then return, the timer no longer resets to zero seconds but instead displays the precise amount of time that has passed.

Other than that looks good to me.

Comment (JV0812) [reply blog-service/2023-09-08-search.md]: Thanks @manashar4. I have updated this, let me ask for a quick approval on this PR from my team.

File: blog-service/2023-09-08-search.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  - search
+  - query
+  - elapsed timer
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+Bug fix - We've enhanced the elapsed query timer so that while a query is running and your users navigate to a different Sumo internal tab or browser tab and then return, the timer no longer resets to zero seconds but instead displays the precise amount of time that has passed.
```
Comment (jpipkin1) [inline blog-service/2023-09-08-search.md:14]:
```suggestion
Bug fix - We've enhanced the elapsed query timer so that while a query is running, and your users navigate to a different Sumo internal tab or browser tab and then return, the timer no longer resets to zero seconds but instead displays the precise amount of time that has passed.
```

**Review by jpipkin1 — APPROVED** [review]
Looks good, @JV0812. I just suggest adding a comma.
